schedule/virt_autotest/slem_ppc_slem_virtualization.yaml

```diff
@@ -0,0 +1,12 @@
+name:           slem_virtualization
+description:    >
+    Maintainer: xguo@suse.com, qe-virt@suse.de
+    SLE Micro as virtualization host. It runs qemu as emulator and uses default raw image.
+schedule:
+    - microos/disk_boot
+    - transactional/host_config
+    - console/suseconnect_scc
+    - qemu/info
+    - qemu/qemu
+    - console/journal_check
+    - shutdown/shutdown
```

Comment:
Any different modules compared to other arches ? I think you can reuse existing yaml schedule file. It seems all loaded modules are the same.

Reply:
It seems that all loaded modules are the same, just superficially. Seeing the loaded modules now is a comparison of the results of existing SL Micro6.1 Functional and Virtualization tests. 
Meanwhile, existing SL Micro6.1 openqa test call with main_micro_alp.pm, I prefer to use the newly added yaml method for ppc image test.

Reply:
Find a simpler way to do this instead of using a yaml file